### PR TITLE
Station placed shuttles, cutting down on duplicate zlevels

### PR DESCRIPTION
**Pull request by Jordie0608**
Moves all shuttles so they are mapped on the station .dmm rather than centcom or deep space. This means that maps with shuttles different from box's default don't need to have their own version of centcom just for the shuttles and can be removed thus reducing the repo size by about 3MB and making it quicker to edit maps.

The airlock emergency shuttle dock at centcomm has been replaced with a line of poddoors that open when the shuttle docks at round-end if there isn't empty space next to them. This is so shuttle designs don't have to conform to line up with airlocks.

Tested to work for all maps, there are a few design issues with existing shuttles on some maps but I will leave that up to the respective map makers, none of them are critical.

Also now fixes Asteroidstation not compiling.

### CODE REVIEW

**Comment by xxalpha** [comment]
Great:grey_exclamation:

**Comment by Strathcona** [comment]
Unfortunately DiscStation modifies the actual docking bay at centcom.  Meta also has edits to centcom itself.

**Comment by Jordie0608** [comment]
@Strathcona The change are minor enough to work around.

**Comment by Giacom** [comment]
Seems useless if the maps need to modify the centcom docks anyway.

**Comment by tuypo1** [comment]
does this mean the ai can look through cameras on escape pods now

**Comment by Anonus** [comment]
The only important thing you need to do on dream is not change the direction of the conveyor belts, this is basically the system I was waiting for because it means I won't have to edit z2 at all.

Or in other words, I don't mind if you do the edits as long as A: z2 turns out unedited, meaning dream works with the generic z2 and B: the conveyor belts on the supply shuttle work correctly while on station
(and C: shuttle rotation works, but that's kind of a given)

**Comment by KorPhaeron** [comment]
Should just make a rule against editing centcomm, doesn't seem worth the maintainability issues for something plays see 30 seconds after round end

**Comment by Tokiko1** [comment]
There is a few issues I can see: Differently shaped cargo shuttles not fitting into the default z2 cargo area, horizontal shuttles having a vertical space transition(though you could just rotate them during transit I suppose) and certain escape shuttles having difficulties docking at centcom due to their shapes.

Feel free to edit birdstation, but note that the cargo shuttle has a different shape and the escape shuttle will probably require a new double door docking area at centcom.

**Comment by Jordie0608** [comment]
Finally got around to finishing this. Bring on the merge conflicts.

@Giacom They don't need to be.
@tuypo1 This has nothing to do with the AI or cameras

File: _maps/dreamstation.dm [hunk 1]
```diff
@@ -16,11 +16,10 @@ z7 = empty space
 #if !defined(MAP_FILE)
 
         #include "map_files\DreamStation\dreamstation04.dmm"
-        #include "map_files\DreamStation\z2.dmm"
-        /*z2 is literally only edited because converyor belts don't play nice with shuttles*/
+        #include "map_files\generic\z2.dmm"
         #include "map_files\generic\z3.dmm"
         #include "map_files\generic\z4.dmm"
-        #include "map_files\DreamStation\z5.dmm"
```
Comment (Anonus) [inline _maps/dreamstation.dm]: hey hey this was edited for a reason, specifically making all the unconnected pipe errors stop fucking happening

Comment (Jordie0608) [inline _maps/dreamstation.dm]: Those edits could just be done to the generic z5 non?

Comment (Anonus) [inline _maps/dreamstation.dm]: Oh right, and making sure there isn't a labor camp console, because dream doesn't have a labor camp or shuttle.

**Comment by Anonus** [comment]
I have caused you conflicts, I am sorry.

**Pull request merged**